packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -228,235 +160,41 @@ export const Tree: ComponentWithAs<'div', TreeProps> &
   });
 
   const treeRef = React.useRef<HTMLElement>();
```

Comment:
I don't see any usage of `treeRef`, can we remove it?

Reply:
I think we can.
Originally `treeRef` was only used for finding first child, and it is used to traverse DOM. 
We now uses flat tree to get the id of the first child instead of traversing DOM, so I will remove it.